stats.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"strconv"
+	"sync/atomic"
+)
+
+type Stats struct {
+	producerUnflushed int64
+	producerErr       int64
+}
+
+func (s *Stats) toSlice() []interface{} {
+	return []interface{}{
+		"producer.unflushed.messages",
+		strconv.FormatInt(atomic.LoadInt64(&s.producerUnflushed), 10),
```

Comment:
Can  we use an integer type as the value? the redisproto _should_ support it.

Reply:
Package `atomic` has no `int` alias. We either have to use `int32` or `int64`.